apycula/gowin_pack.py

```diff
@@ -27,6 +28,8 @@ def get_pips(data):
             if res:
                 row, col, src, dest = res.groups()
                 yield int(row), int(col), src, dest
+            elif "ALIAS" in pip:
+                print("Ignoring alias:", pip)
```

Comment:
Huh, did you actually run into aliases with the nextpnr-gowin flow? AFAIK this was only a nextpnr-generic thing.

Reply:
You are right, I added this when using generic and it was bit confusing to see errors.

Reply:
I doubt that would have worked anyway because the primitives have different names. It wouldn't be that hard *at the moment* to support both, until support for more primitives is added.